pkg/karmadactl/get/get.go

```diff
@@ -149,7 +164,9 @@ func NewCmdGet(f util.Factory, parentCommand string, streams genericiooptions.IO
 
 // CommandGetOptions contains the input to the get command.
 type CommandGetOptions struct {
-	Clusters []string
+	Clusters             []string
+	OperationScope       string
```

Comment:
```suggestion
	OperationScope       OperationScope
```
Why not use the real type? So, you don't have to convert it to string.

Reply:
This is because `o.OperationScope` is bound to the flag `--operation-Scope` by using `flags.StringVarP` before. However, I found that we can use `flags.Var` to bind a flag to a custom type. I will correct it as soon as possible.